src/drivers/imu/invensense/icm20602/ICM20602.hpp

```diff
@@ -143,11 +152,9 @@ class ICM20602 : public device::SPI, public px4::ScheduledWorkItem
 		WAIT_FOR_RESET,
 		CONFIGURE,
 		FIFO_READ,
-		REQUEST_STOP,
-		STOPPED,
 	};
 
-	px4::atomic<STATE> _state{STATE::RESET};
+	STATE _state{STATE::RESET};
```

Comment:
Thanks for cleaning this up. At one point I was experimenting with initiating certain state transitions from the interrupt for NuttX SPI_TRIGGER.